src/test/resources/reader/1566413960000

```diff
@@ -5,6 +5,14 @@
 {"DestAddr":"00000000000000000000000000000000","Net_TCP_NumFlows":4,"Net_TCP_TxQ":0.0,"Net_TCP_RxQ":0.0,"Net_TCP_Lost":0.0,"Net_TCP_SendCWND":10.0,"Net_TCP_SSThresh":0.0}
 {"DestAddr":"0000000000000000FFFF0000534ED40A","Net_TCP_NumFlows":26,"Net_TCP_TxQ":0.0,"Net_TCP_RxQ":0.0,"Net_TCP_Lost":0.0,"Net_TCP_SendCWND":14.461538461538462,"Net_TCP_SSThresh":13.153846153846153}
 $
+^shard_state_metrics
+{"current_time":1566413936488}
+{"IndexName":"pmc","ShardID":2,"ShardType":"primary","NodeName":"elasticsearch2","Shard_State_Active":1,"Shard_State_Initializing":0,"Shard_State_Unassigned":0}
```

Comment:
Given a shard can only be in one of the three states, can we use a key like `shard_state` and the value can be the state it is in ?

Reply:
This was done to make it more clear and remove ambiguity of the possible values of Shard State.

Reply:
Changed the whole structure.